File: complete/src/main/java/com/example/integration/IntegrationApplication.java
package com.example.integration;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.ConfigurableApplicationContext;
import org.springframework.context.annotation.ImportResource;

@SpringBootApplication
@ImportResource("/integration/integration.xml")
public class IntegrationApplication {
	public static void main(String[] args) throws Exception {
		ConfigurableApplicationContext ctx = new SpringApplication(IntegrationApplication.class).run(args);
		System.out.println("Hit Enter to terminate");
		System.in.read();
		ctx.close();
	}

}

File: complete-kotlin/src/main/kotlin/com/example/integration/IntegrationApplication.kt
package com.example.integration

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.ImportResource

@SpringBootApplication
@ImportResource("/integration/integration.xml")
class IntegrationApplication

fun main(args: Array<String>) {
    val ctx = runApplication<IntegrationApplication>(*args)
    println("Hit Enter to terminate")
    System.`in`.read()
    ctx.close()
}